Source/AliveLib/Glukkon.hpp

```diff
@@ -92,15 +92,75 @@ class Glukkon;
 using TGlukkonAIFn = __int16 (Glukkon::*)();
 using TGlukkonMotionFn = void (Glukkon::*)();
 
+struct Glukkon_SaveState
+{
+    Types field_0_id;
+    __int16 field_2_pad;
+    __int32 field_4_object_id;
+    FP field_8_xpos;
+    FP field_C_ypos;
+    FP field_10_xvel;
+    FP field_14_yvel;
+    __int16 field_18_path;
+    LevelIds field_1A_level;
+    FP field_1C_sprite_scale;
+    WORD field_20_r;
+    WORD field_22_g;
+    WORD field_24_b;
+    WORD field_26_flipX;
+    WORD field_28_current_motion;
+    WORD field_2A_current_frame;
+    WORD field_2C_frame_change_counter;
+    BYTE field_2E;
+    BYTE field_2F;
+    FP field_30_health;
+    WORD field_34_current_motion;
+    WORD field_36_next_motion;
+    __int16 field_38;
+    char field_3A_line_type;
+    BYTE field_3B_pad;
+    __int32 field_3C_pad;
+    WORD field_40_bIsActiveChar;
+    __int16 field_42_pad;
+    __int32 field_44_tlvInfo;
+    __int32 field_48_brain_state_idx;
+    __int32 field_4C_pad;
+    __int32 field_50;
+    __int32 field_54;
+    FP field_58;
+    __int16 field_5C_pad;
+    __int16 field_5E;
+    LevelIds field_60_level;
+    __int16 field_62_path;
+    __int16 field_64_camera;
+    GlukkonSpeak field_66_speak;
+    __int32 field_68;
+    FP field_6C;
+    __int32 field_70;
+    __int32 field_74;
+    __int32 field_78;
+    __int32 field_7C;
+    __int32 field_80;
+    __int32 field_84;
+    __int32 field_88_obj_id;
+    __int16 field_8C;
+    Types field_8E;
+};
+ALIVE_ASSERT_SIZEOF_ALWAYS(Glukkon_SaveState, 144);
+
 class Glukkon : public BaseAliveGameObject
 {
 public:
     EXPORT Glukkon* ctor_43F030(Path_Glukkon* pTlv, int tlvInfo);
 
     virtual BaseGameObject* VDestructor(signed int flags) override;
 
+    EXPORT static signed int CC Glukkon::CreateFromSaveState_442830(const BYTE* a1);

```

Comment:
the others are int not explicit signed int - param a1 should be renamed to pBuffer